src/main/java/org/mule/extension/api/introspection/ImmutableSourceModel.java

```diff
@@ -46,13 +48,15 @@ public ImmutableSourceModel(String name,
                                 DataType attributesType,
                                 SourceFactory sourceFactory,
                                 Map<String, Object> modelProperties,
-                                List<InterceptorFactory> interceptorFactories)
+                                List<InterceptorFactory> interceptorFactories,
+                                Optional<ExceptionEnricherFactory> exceptionEnricherFactory)
```

Comment:
leave it as originally intended by juani. We're going to be splitting all the model classes soon